src/MCPClient/lib/clientScripts/extract_contents.py

```diff
@@ -28,13 +33,8 @@
 TRANSFER_DIRECTORY = "%transferDirectory%"
 
 
-def temporary_directory(file_path, date):
-    if file_path_cache.get(file_path):
-        return file_path_cache[file_path]
-    else:
-        path = file_path + '-' + date
-        file_path_cache[file_path] = path
-        return path
+def suffix_date(file_path, date):
```

Comment:
The function is pretty self-explanatory, but do you think we should add a docstring as well?